templates/web/quiz/quiz.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'web/layout.html' %}
+
+{% block title %}
+    {{ quiz.title }}
+{% endblock %}
+
+{% block content %}
+    <div id="page-content-wrapper">
+        <div class="content-header">
+            <h2>
+                {{ quiz.title }}
+                {% if is_teacher and scoring is None %}
+                    <a href="/teacher/quiz/{{ quiz.id }}/edit" class="text-muted" title="Edit quiz"><span
+                            class="iconify"
+                            data-icon="fa-solid:pen"></span></a>
+                {% endif %}
+            </h2>
+        </div>
+        <kelvin-quiz quiz_html="{{ quiz_html }}" is_teacher="{{ is_teacher|lower }}"
```

Comment:
It's unfortunate that we would have to pass everything as strings because it is a web component. Please take a look at `student_page.html` to see how we can do it properly by passing the typed props to the Vue component, without going through web components.